src/test/java/fr/cnes/sonar/plugins/icode/exceptions/ICodeExceptionTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * This file is part of sonar-icode-cnes-plugin.
+ *
+ * sonar-icode-cnes-plugin is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sonar-icode-cnes-plugin is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with sonar-icode-cnes-plugin.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package fr.cnes.sonar.plugins.icode.exceptions;
+
+import org.junit.Assert;
+
+public class ICodeExceptionTest {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add some tests to this class. ([squid:S2187](https://sonarcloud.io/organizations/lequal/rules#rule_key=squid%3AS2187))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fr.cnes.sonarqube.plugins%3Asonaricode&pullRequest=37&issues=AWXs1igg2IDoRcZ5vwWn&open=AWXs1igg2IDoRcZ5vwWn)

[ISSUE_KEY]: AWXs1igg2IDoRcZ5vwWn